docs/core/entity.md

```diff
@@ -8,22 +8,22 @@ For a generic introduction of entities, see [entities architecture](../architect
 ## Basic implementation
 
 Below is an example switch entity that keeps track of its state in memory.
+In addition, the switch in the example represents the main feature of a device,
+meaning it has the same name as its device.
+
+Please refer to [Entity naming](#entity-naming) for how to give an entity its own name.
 
 ```python
 from homeassistant.components.switch import SwitchEntity
 
 
 class MySwitch(SwitchEntity):
     _attr_has_entity_name = True
+    _attr_name = False
```

Comment:
The default is `None` so we can remove this.
```suggestion
```

Reply:
Should we set a unique_id too? Otherwise the new naming feature won't activate.

Reply:
Not only is a `unique_id` needed, the entity also needs to be tied to a device one way or the other.

Maybe we can mention in the note that the example is not complete and add a link to the device registry documentation?

Reply:
Sound good.